docs/examples/KModal/ModalWithSubmit&Cancel.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+
+  <div>
+    <KButton @click="openModal">Click Me</KButton>
+    <p v-if="showMessage">Successfully submitted the Form</p>
+    <KModal
+      v-if="showModal"
+      :appendToOverlay="true"
+      size="medium"
+      title="Submit Form"
+      submitText="Submit"
+      :submitDisabled="!enableSubmit"
+      cancelText="Cancel"
+      @submit="modalEmits('submit')"
+      @cancel="modalEmits('cancel')"
+    >
+      <template>
+        <p>Check the below check box to enable the submit button</p>
+        <KCheckbox
+          :checked="enableSubmit"
+          label="Enable Submit"
```

Comment:
```suggestion
          label="Enable submit"
```

Aligns case to writing guidelines.